tests/runtest.cmd

```diff
@@ -111,6 +112,7 @@ if /i "%1" == "link"                  (set DoLink=true&set ILLINK=%2&shift&shift
 if /i "%1" == "tieredcompilation"     (set COMPLUS_EXPERIMENTAL_TieredCompilation=1&shift&goto Arg_Loop)
 if /i "%1" == "gcname"                (set COMPlus_GCName=%2&shift&shift&goto Arg_Loop)
 if /i "%1" == "timeout"               (set __TestTimeout=%2&shift&shift&goto Arg_Loop)
+if /i "%1" == "userassemblies"        (set __UserTestAssemblies=%2&shift&shift&goto Arg_Loop) 
```

Comment:
`userassemblies` switch followed by test assembly unique name fragment can be used to select tests to run. Globbing patterns respected by MSBuild are supported as well. For example command `tests\runtest x64 Checked userassemblies JIT.*` will run the following group of tests:

```

  === TEST EXECUTION SUMMARY ===
     JIT.CheckProjects.XUnitWrapper        Total:    1, Errors: 0, Failed: 0, Skipped: 0, Time:  209,216s
     JIT.CodeGenBringUpTests.XUnitWrapper  Total:    8, Errors: 0, Failed: 0, Skipped: 0, Time:   25,009s
     JIT.Directed.XUnitWrapper             Total:  122, Errors: 0, Failed: 0, Skipped: 0, Time:  289,984s
     JIT.Generics.XUnitWrapper             Total:   40, Errors: 0, Failed: 0, Skipped: 0, Time:  339,441s
     JIT.HardwareIntrinsics.XUnitWrapper   Total:  258, Errors: 0, Failed: 0, Skipped: 0, Time:  365,856s
     JIT.IL_Conformance.XUnitWrapper       Total:  104, Errors: 0, Failed: 0, Skipped: 0, Time:  253,175s
     JIT.Intrinsics.XUnitWrapper           Total:   12, Errors: 0, Failed: 0, Skipped: 0, Time:  215,758s
     JIT.jit64.XUnitWrapper                Total:  108, Errors: 0, Failed: 0, Skipped: 0, Time:  334,587s
     JIT.Methodical.XUnitWrapper           Total:  754, Errors: 0, Failed: 0, Skipped: 0, Time:  503,582s
     JIT.opt.XUnitWrapper                  Total:   21, Errors: 0, Failed: 0, Skipped: 0, Time:   94,342s
     JIT.Performance.XUnitWrapper          Total:   83, Errors: 0, Failed: 0, Skipped: 0, Time:  236,479s
     JIT.Regression.XUnitWrapper           Total:  442, Errors: 0, Failed: 0, Skipped: 0, Time:  408,550s
     JIT.RyuJIT.XUnitWrapper               Total:    1, Errors: 0, Failed: 0, Skipped: 0, Time:   34,365s
     JIT.SIMD.XUnitWrapper                 Total:   97, Errors: 0, Failed: 0, Skipped: 0, Time:  224,902s
     JIT.superpmi.XUnitWrapper             Total:    1, Errors: 0, Failed: 0, Skipped: 0, Time:  245,802s
                                                  ----          -          -           -        ---------
                                     GRAND TOTAL: 2052          0          0           0        3781,047s (517,394s)

Build succeeded.
    0 Warning(s)
    0 Error(s)

Time Elapsed 00:08:38.19
RUNTEST: Test run successful. Refer to the log files for details:
    e:\src\ms\dotnet\coreclr-sse2\tests\..\bin\Logs\TestRun_Windows_NT__x64__Checked.html
    e:\src\ms\dotnet\coreclr-sse2\tests\..\bin\Logs\TestRun_Windows_NT__x64__Checked.xml

```